proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -136,6 +138,12 @@ service PluginService {
   // a plugin may cause breaking changes for templates using that plugin,
   // and should be done with extreme care.
   rpc DeletePlugin(DeletePluginRequest) returns (DeletePluginResponse);
+  // AddPluginContributor add a role to an user in the plugin.
+  rpc AddPluginContributor(AddPluginContributorRequest) returns (AddPluginContributorResponse);
+  // UpdatePluginContributor update the user's contributorship information in the plugin.
+  rpc UpdatePluginContributor(UpdatePluginContributorRequest) returns (UpdatePluginContributorResponse);
+  // RemovePluginContributor remove the role of an user in the plugin.
+  rpc RemovePluginContributor(RemovePluginContributorRequest) returns (RemovePluginContributorResponse);
```

Comment:
We could do a similar things as we discussed for recommended repositories here; `SetUserPluginRole`, which does both add, update and remove (set to UNSPECIFIED). What do you think?

Reply:
my only concern is the logic behind this is probably much more complicated than recommendation, so group them together might increase the complexity of implementation too much, which I guess is also something we want to avoid? there is some trade off and I'm not really sure which one is better

Reply:
I would think it shouldn't be much harder to use, but would be good to get @timostamm to weigh in.

Reply:
There will be likely be three separate call sites in the frontend, I don't see any complication on this side if the three RPCs were rolled together.

Reply:
cool, will update to group them together then